services/tinlake/actions.ts

```diff
@@ -313,205 +300,151 @@ export async function borrow(tinlake: any, loan: Loan, amount: string) {
   }
 
   // borrow with proxy
-  let borrowRes
   try {
-    borrowRes = await tinlake.proxyLockBorrowWithdraw(proxy, loanId, amount, address)
+    return await tinlake.proxyLockBorrowWithdraw(proxy.toString(), loanId, amount, address!)
   } catch (e) {
     return loggedError(e, 'Could not finance asset.', loanId)
```

Comment:
Why do we sometimes still use `loggedError`, and sometimes not?

Reply:
Where don't we use it? I think it's always used, right?